05-geometry-operations.qmd

```diff
@@ -502,7 +505,155 @@ shapely.geometry.MultiPoint(polygon.exterior.coords)
 
 ### Aggregation and disaggregation
 
-...
+Raster datasets can also differ with regard to their resolution. 
```

Comment:
Not only can they, they do. How about something like

> Raster datasets vary based on their resolution, from high resolution datasets that enable individual trees to be seen, to low resolution datasets covering the whole Earth.

Reply:
Great! Added it